ci/docker/Dockerfile.build.armv7

```diff
@@ -20,13 +20,13 @@
 
 FROM dockcross/linux-armv7
 
-ENV ARCH armv71
+ENV ARCH armv7
 ENV CC /usr/bin/arm-linux-gnueabihf-gcc
 ENV CXX /usr/bin/arm-linux-gnueabihf-g++
 
-RUN apt-get update && \
-    apt-get install -y libopenblas-dev:armhf && \
-    rm -rf /var/lib/apt/lists/*
+RUN apt-get update
+RUN apt-get install -y libopenblas-dev:armhf unzip
```

Comment:
Let's build openblas ouself like for other arm platforms?

Reply:
sure, I would make another PR for that.